### ISSUE TITLE
where is init-elpa.el file

### ISSUE THREAD

**Issue by cjack** [redguardtoo/emacs.d#379]
Forgive me if this is a silly issue.
When I was trying the instruction of "Uncomment below code in init-elpa.el and start Emacs now!", I cannot find the location of  "init-elpa.el" file. I have unzip the "https://github.com/redguardtoo/myelpa/archive/2.0.zip" into the "~/projs/myelpa".

**Comment by redguardtoo** [comment]
`~/.emacs.d/lisp/init-elpa.el`,
myelpa is is basically packages repository, you don't need do anything on it after extraction, then you need modify you setup (init-elpa.el) to to tell emacs you use the local repository (the ~/projs/myelpa) instead of the online repository http://elpa.gnu.org or http://melpa.org

**Comment by cjack** [comment]
Many thanks. I found it and uncommented the code. 
But I also found the code as follows,
<img width="571" alt="screen shot 2015-11-16 at 2 19 35 pm" src="https://cloud.githubusercontent.com/assets/4144516/11173647/1e60bb26-8c6d-11e5-8698-fd5ea73e8939.png">
As you mentioned, we want to use the local repository instead of the online ones. Should I comment those codes, or it is OK to just leave them? Not sure about the Lisp grammar, but I guess it would be ok to just leave them. 
Not an emergency issue, thanks for your patience.
